etc/tinyproxy.conf.in

```diff
@@ -56,8 +56,8 @@ Timeout 600
 #   /usr/share/tinyproxy
 #   /etc/tinyproxy
 #
-#ErrorFile 404 "@pkgdatadir@/404.html"
```

Comment:
are there no other cases in the code where a 404 is returned ?

Reply:
At least I couldn't find any, I grepped for `404`